src/components/pages/profile/[name]/Profile.tsx

```diff
@@ -182,8 +182,10 @@ const ProfileContent = ({ isSelf, isLoading: parentIsLoading, name }: Props) =>
       name,
       decodedName: profile?.decodedName,
       normalisedName,
+      tabs,
+      tab,
```

Comment:
Pass in "visibleTabs" instead of just tabs. "visibleTabs" will filter out the permissions tab for legacy names.  This may cause a dependency array error. Let me know if it does and I can help you fix it.